pkg/vsphere/session/errors.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package session
+
+import "fmt"
+
+// SDKURLError is returned when the soap SDK URL cannot be parsed
+type SDKURLError struct {
+	Service string
+	Err     error
```

Comment:
I've been running into issues with the way `fmt.Sprintf` handles anonymous `error` fields as arguments. I'll defer this change for now and look for the exact problem.